### PR TITLE
Docs: Simplify `prod:serve`

### PR DESCRIPTION
**Pull request by ryan-roemer**
This is way simpler at the cost of being linux/mac only because it's a symlink.

@boygirl -- assuming you're on board, we should have this be the "official" way and port it back to spectacle.

@parkerziegler -- I got ninety nine problems but the badge ain't one (if you want to try it out)

### CODE REVIEW

File: docs/dist-prod/open-source/renature [hunk 1]
```diff
@@ -0,0 +1 @@
+../../dist
```
Comment (ryan-roemer) [inline docs/dist-prod/open-source/renature]: This is a symlink

**Comment by ryan-roemer** [comment]
I'm not sure why `yarn.lock` changed (?)

File: docs/scripts/deploy/aws.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const EXECA_OPTS = {
 };
 
 const { log } = console;
-const logMsg = msg => log(chalk`[{cyan deploy/surge}] ${msg}`);
+const logMsg = msg => log(chalk`[{cyan deploy/aws}] ${msg}`);
```
Comment (ryan-roemer) [inline docs/scripts/deploy/aws.js]: Something I noticed from earlier...

**Comment by ryan-roemer** [comment]
Staging: https://formidable-com-renature-staging-25.surge.sh/open-source/renature/

File: docs/dist-prod/open-source/renature (hunk 1 above)
Comment (parkerziegler) [reply docs/dist-prod/open-source/renature]: Ah interesting, so this gets around the rewrites altogether and places everything at `/open-source/renature`? What is it symlinking to (mostly just curious)?

**Comment by boygirl** [comment]
@ryan-roemer That works for me! Much more straightforward

File: docs/dist-prod/open-source/renature (hunk 1 above)
Comment (ryan-roemer) [reply docs/dist-prod/open-source/renature]: There are a lot of `/ROOT_PATH` links throughout that we can't easily relativize. So when we build for production, root is `/open-source/renature`. When in dev it's `/`

So we need to account for that when previewing. The existing way for a prod build which has "root is `/open-source/renature`" assumption built in was server-side request rewrites. But this has issues like `@` getting encoded and 404-ing badge images, etc.

The simpler solution here is to build for `/open-source/renature` and then just **serve it** from `/open-source/renature` 😛 😂  and that's what this PR is...

**Pull request merged**